dotfiles/waybar/modules.json

```diff
@@ -100,7 +100,7 @@
         "return-type": "json",       
         "exec": "~/dotfiles/scripts/updates.sh",
         "restart-interval": 60,
-        "on-click": "alacritty --class dotfiles-floating -e ~/dotfiles/scripts/installupdates.sh",
+        "on-click": "$TERM --class dotfiles-floating -e ~/dotfiles/scripts/installupdates.sh",
```

Comment:
Can you replace $TERM with $(cat ~/dotfiles/.settings/terminal.sh). Then it will use the settings from the ML4W Dotfiles App (Tab System)